sources/packages/backend/apps/db-migrations/src/sql/ApplicationRestrictionBypasses/Create-application-restriction-bypasses.sql

```diff
@@ -0,0 +1,51 @@
+CREATE TABLE sims.application_restriction_bypasses (
+  id SERIAL PRIMARY KEY,
+  application_id INT NOT NULL REFERENCES sims.applications(id),
+  student_restriction_id INT NOT NULL REFERENCES sims.student_restrictions(id),
+  bypass_behavior sims.restriction_bypass_behaviors NOT NULL,
+  is_active BOOLEAN NOT NULL,
+  creation_note_id INT NOT NULL REFERENCES sims.notes(id),
+  bypass_created_by INT NOT NULL REFERENCES sims.users(id),
+  bypass_created_date TIMESTAMP WITH TIME ZONE NOT NULL,
```

Comment:
Thinking twice, I am not able to particularly see a difference between CreatedAt and CreatedBy and these fields or any benefit unless I am missing something.

Reply:
The idea was just to keep the "table" audit separated from the bypass auditing trying to follow what we did in the past.